corehq/apps/reports/templates/reports/form/partials/single_form.html

```diff
@@ -41,10 +41,25 @@
 </style>
 <script>
     $(function () {
+        function _analytics(action, callback) {
+            var label = 'standalone_form', extra = {};
+            {% if context_case_id %}
```

Comment:
is fine if it was too annoying to do differently but i generally try not to mix javascript and django templates except to initialize variables.